pages/02.administer/15.admin_guide/20.users/05.groups_and_permissions/groups_and_permissions.md

```diff
@@ -18,6 +18,14 @@ The group mechanism can be used to define groups of users which then can be used
 
 Using groups is however useful for semantics, for example if you host multiple groups of friends, associations or businesses on your server, you might want to create groups like `association1` and `association2` and add members of each association to the relevant group.
 
+
+The user group can use a *group list* to send mails to multiple user. This feature is only available with the cli.
+To show all information and mailing alias about a specific group : 
+```shell
+yunohost user group info group_name
+```
```

Comment:
It's also possible to define mail aliases for a group, such that mails sent to `group@domain.tld` will be dispatched to all members of the group.